core/commands/VersionGetCommand.php

```diff
@@ -40,6 +40,27 @@ class VersionGetCommand extends Command {
 	 * @var integer|null
 	 */
 	private $version_id;
+	
+	/**
+	 * The released versions will be returned.
+	 *
+	 * @var boolean
+	 */
+	private $r_released;
+	
+	/**
+	 * The obsolete versions will be returned.
+	 *
+	 * @var boolean
+	 */
+	private $r_obsolete;
+	
+	/**
+	 * The inherited versions will be returned.
+	 *
+	 * @var boolean
+	 */
+	private $r_inherited;
```

Comment:
I don't understand the purpose of these new properties, they are never assigned.